content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,64 @@
+# Machine Learning Model Registry
+
+A **model registry** is a repository to catalog ML models and their versions.
+Artifacts from your data science projects can be discovered, audited, shared,
```

Comment:
Let's keep it simple - Models, not artifacts

Reply:
OK removed most "artifact" mentions now.